core/App/screens/CredentialDetails.tsx

```diff
@@ -141,30 +207,113 @@ const CredentialDetails: React.FC<CredentialDetailsProps> = ({ navigation, route
     setIsRemoveModalDisplayed(false)
   }
 
+  const handleDismissRevokedMessage = () => {
+    setIsRevokedMessageHidden(true)
+  }
+
   const callOnRemove = useCallback(() => handleOnRemove(), [])
   const callSubmitRemove = useCallback(() => handleSubmitRemove(), [])
   const callCancelRemove = useCallback(() => handleCancelRemove(), [])
+  const callDismissRevokedMessage = useCallback(() => handleDismissRevokedMessage(), [])
 
-  const header = () => {
+  const renderCredentialCardLogo = () => {
+    return (
+      <View style={styles.logoContainer}>
+        {overlay.cardLayoutOverlay?.logo?.src ? (
+          <Image
+            source={toImageSource(overlay.cardLayoutOverlay?.logo.src)}
+            style={{
+              resizeMode: 'center',
+            }}
+          />
+        ) : (
+          <Text style={[TextTheme.title, { fontSize: 0.5 * logoHeight }]}>
+            {(overlay.metaOverlay?.issuerName || overlay.metaOverlay?.name || 'C')?.charAt(0).toUpperCase()}
+          </Text>
+        )}
+      </View>
+    )
+  }
+
+  const renderCredentialDetailPrimaryHeader = () => {
```

Comment:
I think the word `render` in RN is meant to indicate converting a component to its HTML equivalent. Might be more clear to drop the word here and just use `credentialDetailPrimaryHeader` since its not rendering. A bit more consistent also since "header" below does the same thing but does not use the word `render`

Reply:
Agreed. Will drop the term in favour of the component name